frontend-cert/js-projects/dates-lab/user-stories.md

```diff
@@ -0,0 +1,12 @@
+1. You should create a variable named `currentDate` using `let` and assign it the current date and time using the `Date` object.
+1. You should create a function named `formatDateMMDDYYYY` that takes a date object as a parameter and returns the date formatted as `MM/DD/YYYY`.
+1. Inside the `formatDateMMDDYYYY` function, you should create variables named `month`, `day`, and `year` using `let`.
+1. The `month` variable should be assigned a String value of the month from the date object using the `getMonth` method and adding 1 to the result, and a `padStart` method to ensure the month is always two digits.
```

Comment:
same issue here.
`padStart` and `padEnd` is not taught.

if you think we should teach it, then we can add a video for it.
otherwise, you will need to update this user story

Reply:
Okay, will update the user stories rather